third_party/rust/third_party.toml

```diff
@@ -135,8 +135,10 @@ lazy_static = "1.4.0"
 # regex = "1.5.5"
 
 # brave-wallet
+bech32 = "0.9.1"
```

Comment:
there should normally also be changes to Cargo.toml/Cargo.lock here, but atm things are in a weird state so this is ok